README.md

```diff
@@ -24,4 +24,9 @@ To create all necessary tables in database, run:
 ## Code formatting
 Before committing, format code using `black` formater:
 
-```black .```
\ No newline at end of file
+```black .```
+
+## Text extraction from image
+Download and install tesseract (https://digi.bib.uni-mannheim.de/tesseract/).
```

Comment:
### Installation

 - *Windows*:

   Download the precompiled file (_tesseract-ocr-w64-setup-v4.1.0.20190314.exe_) and install tesseract from 
   here : [https://digi.bib.uni-mannheim.de/tesseract/](https://digi.bib.uni-mannheim.de/tesseract/).
    Don't forget to place the installation path into PATH environment variable.

 - *Linux (Ubuntu, debian)*: 
   ```
   sudo apt-get install tesseract-ocr
   ```
 - *MacOS*:
   ```
   brew install tesseract
   ```